twitter/twitter.php

```diff
@@ -1466,23 +1452,85 @@ function twitter_expand_entities($body, stdClass $status, $picture)
 
 	// Footer will be taken care of with a share block in the case of a quote
 	if (empty($status->quoted_status)) {
-		$footer = '';
-		if ($attachmentUrl) {
-			$footer = "\n" . PageInfo::getFooterFromUrl($attachmentUrl, false, $picture);
-		}
-
-		if (trim($footer)) {
-			$body .= $footer;
-		} elseif ($picture) {
+		if ($picture) {
 			$body .= "\n\n[img]" . $picture . "[/img]\n";
-		} else {
-			$body = PageInfo::searchAndAppendToBody($body);
 		}
 	}
 
 	return ['body' => trim($body), 'plain' => trim($plain), 'taglist' => $taglist];
 }
 
+/**
+ * Store entity attachments
+ *
+ * @param integer $uriid
+ * @param object $post Twitter object with the post
+ */
+function twitter_store_attachments(int $uriid, $post)
+{
+	if (!empty($post->extended_entities->media)) {
+		foreach ($post->extended_entities->media AS $medium) {
+			switch ($medium->type) {
+				case 'photo':
+					$attachment = ['uri-id' => $uriid, 'type' => Post\Media::IMAGE];
+
+					// Later store the large picture when we don't add them to the body anymore
```

Comment:
I don't understand this comment?

Reply:
Me too.

Reply:
What does "later" mean here? Later in the code or is it a `@TODO`?

Reply:
It's a `@TODO`